package.json

```diff
@@ -34,6 +34,19 @@
     "eslint-plugin-prettier": "^4.0.0",
     "lerna": "^4.0.0",
     "prettier": "^2.4.1",
-    "typescript": "^4.5.2"
+    "ts-node": "^10.4.0",
+    "@types/chai": "^4.2.21",
+    "@types/mocha": "^9.0.0",
```

Comment:
We use `tape` in this repository to write tests, please do not introduce another testing framework. Nor do we need `chai`.
Also I do not see tests anywhere, so not sure why this would be needed?